.github/workflows/bundle-size.yml

```diff
@@ -23,15 +23,15 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v4
         with:
-          node-version: '20'
+          node-version-file: .node_version
 
       - name: Setup pnpm
         uses: pnpm/action-setup@v4
         with:
           version: 10.15.0
 
       - name: Install dependencies
-        run: pnpm install --frozen-lockfile
+        run: rm -rf node_modules && pnpm install --force
```

Comment:
Very opposed to stop using `--frozen-lockfile`. I need to be convinced.

Reply:
It was done to fix a build. For some reason (weird Vercel caching kinda) only this solved the issue.
I will try to get it back.